scholarx/archive/2020/mentees.json

```diff
@@ -0,0 +1,426 @@
+    },
+
+    {
+        "name": "Shehana Iqbal",
+        "image": "shehana.jpg",
+        "mentor": "Jeewika Ranaweera",
+        "university": "University of Moratuwa ",
+        "linkedin": "https://www.linkedin.com/in/shehana-iqbal/"
+    },
+
+    {
+        "name": "Asel Peiris",
+        "image": "asel.jpg",
+        "mentor": "Lahiru Maramba",
+        "university": "Sri Lanka Institute of Information Technology",
+        "linkedin": "https://www.linkedin.com/in/asel-peiris-76536158/"
+    },
+
+    {
+        "name": "Mithunjha Anandakumar",
+        "image": "mithunjha.jpg",
+        "mentor": "Dinithi Sumanaweera",
+        "university": "University of Moratuwa",
+        "linkedin": "https://www.linkedin.com/in/mithunjha-anandakumar-441618184/"
+    },
+
+    {
+        "name": "Shasika Rajapaksha",
+        "image": "shasika.jpg",
+        "mentor": "Dr. Nadeera Nilupamali",
+        "university": "University of Moratuwa",
+        "linkedin": "https://www.linkedin.com/in/shasika-udayanga-95406815b/"
+    },
+
+    {
+        "name": "Dineth Mudalige",
+        "image": "dineth.jpg",
+        "mentor": "Dr. Nadeera Nilupamali",
+        "university": "University of Moratuwa",
+        "linkedin": "https://www.linkedin.com/in/dineth-mudalige-604b09132/"
+    },
+
+    {
+        "name": "Fahim Aslam",
+        "image": "fahim.jpg",
+        "mentor": "Dr. Nadeera Nilupamali",
+        "university": "University of Colombo",
+        "linkedin": "https://www.linkedin.com/in/fahimaslam08/"
+    },
+
+    {
+        "name": "Windhula Gammanpila",
+        "image": "windhula.jpg",
+        "mentor": "Dr. Nadeesha Chandrasena",
+        "university": "University of Sri Jayawardenapura",
+        "linkedin": "https://www.linkedin.com/in/windhula-gammanpila-6a8264151/"
+    },
+
+    {
+        "name": "Muhammed Imdaad",
+        "image": "muhammed.jpg",
+        "mentor": "Dr. Nadeesha Chandrasena",
+        "university": "University of Moratuwa",
+        "linkedin": "https://www.linkedin.com/in/muhammed-imdaad-23356a17b/"
+    },
+
+    {
+        "name": "Ranjula Hettiarachchi",
+        "image": "ranjula.jpg",
+        "mentor": "Dr. Nadeesha Chandrasena",
+        "university": "University of Moratuwa",
+        "linkedin": "https://www.linkedin.com/in/ranjula-hettiarachchi/"
+    },
+
+    {
+        "name": "Dedunu Karunarathne",
+        "image": "dedunu.jpg",
+        "mentor": "Dr. Nadeesha Chandrasena",
+        "university": "University of Moratuwa",
+        "linkedin": "https://www.linkedin.com/in/dedunu-karunarathne/"
+    },
+
+    {
+        "name": "Suchitha Dehigaspitiya",
+        "image": "suchitha.jpg",
+        "mentor": "Dr. Nadeesha Chandrasena",
+        "university": "University of Moratuwa",
+        "linkedin": "https://www.linkedin.com/in/suchitha-dehigaspitiya/"
+    },
+
+    {
+        "name": "Sahan Samuditha",
+        "image": "maleplaceholder.jpeg",
+        "mentor": "Nishanta Rajakaruna",
```

Comment:
It is Prof. Nishanta Rajakaruna. [Apply for all the entries and please check spellings.]